xray/README.md

```diff
@@ -69,6 +69,12 @@ The application is executed with the `run.py` script. I takes some arguments, al
 
 - **--nobuild-xray**: whether or not to build `xray` before running. Default is to build `xray`
 
+- **--save-output**: save the analysis charts in `results` folder.
+
+- **--ascii**: output the analysis chart as text graph.
+
+- **--disable-drop-privileges**: pass the `disable-drop-privileges` flag to `neptun-cli` and `boringtun-cli`.
```

Comment:
Is this something that needs to be passed through as a cli arg or can we just pass this unconditionally to neptun/boringtun?

Reply:
Good question, I guess it depends if we would ever want to run it without `disable-drop-privileges` and how it impacts the tests.

Reply:
Let's leave it like this for now and we can investigate it at a later point if necessary